tests/unittests/OperatorTest.cpp

```diff
@@ -7976,6 +7976,71 @@ TEST_P(OperatorTest, FusedRowwiseQuantizedSparseLengthsWeightedSum_Float) {
       bindings_, mod_, F_, EE_, ElemKind::UInt8FusedQTy, 0.0001);
 }
 
+/// Test Fused-RWQ-SLWS in where the weights are in Fp16, inputs are
+/// UInt8FusedQTy.
+TEST_P(OperatorTest,
```

Comment:
This looks like an almost exact copy of `FusedRowwiseQuantizedSparseLengthsWeightedSum_ConvertedFloat16` -- can we create a helper out of the logic from test, that takes a bool for setting `cctx.precisionConfig.convertFusedToFP16`? E.g. something similar to `testFusedRowwiseQuantizedSparseLengthsWeightedSum`. Then we can reuse the logic for both tests.